lib/SparkPost/Transmission.php

```diff
@@ -63,12 +63,22 @@ private function __construct() {}
 	 */
 	private static function getHttpClient() {
 		if(!isset(self::$request)) {
-			self::$request = new Client();
+			$hostConfig = SparkPost::getConfig();
+			$defaultOptions = [
+				'base_uri'=>self::getBaseUrl($hostConfig),
+				'headers' =>[
+					'Authorization' => $hostConfig['key'],
+					'Content-Type' => 'application/json',
+					'User-Agent' => 'SparkPost PHP SDK v0.1.2'
```

Comment:
I think we should align them across libraries eventually.  It could give us a lot more insight into usage of libraries. I agree with 0.2.0.  From what I've seen, most packages follow a pattern similar to semver for publishing but they don't really call it that.